examples/visual-tests/amp-story-player/player-local-story.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Story Player (Non-AMP)</title>
+    <script>
+      // TODO(): Remove this once unit tests work.
+      __AMP_VISUAL_TEST = true
+    </script>
+    <script async src="../../../dist/amp-story-player-v0.js"></script>
+    <link href="../../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+    <style page>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <style injected>
+      .story-player-iframe {
+        background-size: cover;
+        display: block;
+        position: relative;
+        height: 100%;
+        width: 100%;
+        flex: 0 0 100%;
+      }
+
+      .title {
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 1));
+        display: block;
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        color: #fff;
+        font-size: 40px;
+        padding: 88px 32px 32px 32px;
+        line-height: 1.25;
+        text-shadow: 3px 3px #000;
+      }
+
+      /* TODO: Remove this when poster image is no longer displayed. */
+      amp-story-player a:first-of-type {
+        display: none;
+      }
+    </style>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+    <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras viverra neque ex, sit amet varius sem maximus sed. Suspendisse potenti. Donec erat purus, sagittis sit amet tincidunt ut, maximus sit amet massa. Maecenas venenatis fringilla dui vitae vestibulum. Fusce imperdiet euismod lobortis. Nullam sagittis nunc at tristique mattis. In sodales consectetur mollis. Maecenas sollicitudin, ex vel tempor rutrum, turpis eros interdum enim, sit amet volutpat tortor dolor at ipsum. Nam posuere velit vel urna vulputate interdum. Aenean eu vulputate lorem. Praesent nec nunc sodales, egestas orci sed, hendrerit mauris. Ut blandit turpis non erat sagittis, quis fermentum odio feugiat.</div>
+    <amp-story-player style="width: 360px; height: 600px;">
+      <a href="/examples/visual-tests/amp-story/amp-story-grid-layer-template-fill.html"
```

Comment:
Just thought of this, but can we try using a story that doesn't load an image? I think this might shave off some loading time. An example would be `/examples/visual-tests/amp-story/amp-story-grid-layer-template-horizontal.html`

Reply:
Will try this!

Reply:
Ends up needing the same amount of time or else it is also flaky.